src/app/_services/api.service.spec.ts

```diff
@@ -22,26 +29,159 @@ describe('ApiService', () => {
     // Inject the http service and test controller for each test
     httpClient = TestBed.get(HttpClient);
     httpTestingController = TestBed.get(HttpTestingController);
-    service = new ApiService(httpClient);
+    service = TestBed.get(ApiService);
   });
 
   afterEach(() => {
     // After every test, assert that there are no more pending requests.
     httpTestingController.verify();
   });
 
-  it('#getHttpHeaders should return httpHeaders json object ', () => {
+  it('#api.getHttpHeaders should be defined ', () => {
     expect(service.getHttpHeaders()).toBeDefined();
   });
 
-  it('#getHttpHeadersWithBase64Authorization should return httpHeaders json object ', () => {
-    const paramUsername = "username";
-    const paramPassword = "password";
-    expect(service.getHttpHeadersWithBase64Authorization(paramUsername, paramPassword)).toBeDefined();
+  it('#api.getHttpHeadersWithBase64Authorization should be defined ', () => {
+    expect(service.getHttpHeadersWithBase64Authorization(testUsername, testPassword)).toBeDefined();
+  });
+
+  it('#api.get should be defined ', () => {
+    expect(service.get(testFackUrl, testUsername, testPassword)).toBeDefined();
+  });
+
+  it('#api.get can test HttpClient.get', () => {
+  //  const testFackData = {name: 'Test Fack Data'};
+    const testHeader = service.getHttpHeadersWithBase64Authorization(testUsername, testPassword);
+    // Make an HTTP GET request
+    httpClient.get<any>(testBaseFackUrl, testHeader)
+      .subscribe(data =>
```

Comment:
The comments I made on #30 re: unit tests, apply here too. Basically, your tests are currently testing Angular's `HTTPClient`. We need to test `api.service`.